src/test/java/pl/wedding/management/service/user/InMemoryRepositoryTest.java

```diff
@@ -0,0 +1,68 @@
+package pl.wedding.management.service.user;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import pl.wedding.management.service.user.model.EMail;
+import pl.wedding.management.service.user.model.User;
+
+import static org.assertj.core.api.Assertions.*;
+
+class InMemoryRepositoryTest {
+
+    private static final char[] DUMMY_PASSWORD = new char[]{'p', 'a', 's', 's', 'w', 'o', 'r', 'd'};
+    private final User constUser = createUser("someEmail@dummy.pl");
```

Comment:
Please use 'example.com' domain. It is it's purpose. dummy.pl is registered domain:
```
DOMAIN NAME:           dummy.pl 
registrant type:       individual 
nameservers:           dns111.ovh.net.  
                       ns111.ovh.net.  
created:               2020.03.30 12:02:19 
last modified:         2020.03.30 13:43:09 
renewal date:          2021.03.30 12:02:19 
```

```suggestion
    private final User constUser = createUser("someEmail@example.com");
```

Reply:
Does it matter? It could be even gmail. Whatever:D

Reply:
It is someone's mail. If application will send test email in the future, this email could be spammed. If you think it doesn't matter, please provide here your email. And I'll do some test stress tests of mail server. It's best practice to avoid accidents that may happen in the future, like sending test mailing to whole database of customers. And you can avoid that by for ex. not using real domain addresses, not cursing in test mailing, assuming that everything you write in test may be retrieved by customer, and regime oneself to provide such test data that wouldn't discredit company.

Reply:
That's interesting!:D How could someone use private constant which is placed in "InMemoryRepositoryTest" to do some stress tests on mail server or send someone a real email? That's 100% impossible.  :)

Reply:
It doesn't matter that **now** it is not. You cannot predict how will code evolve, who will be "inspired" by your solution. It is in code, and therefore can in future be part of some major failure. Seen such failures that occurred after few years and code that dev wrote curses in code, because "this could never" go to ... ; it did. Once product owner got message "Something fucked up". Wasn't pleased. 
Somewhat related, seen better, but can't find it now: https://www.commitstrip.com/wp-content/uploads/2013/01/Strips-Fucking-title-550-finalenglish.jpg

Reply:
I'll replace this line cause fighting about this is unnecessary. But such private constats in test class which includes only add/remove operations on HashSet are harmless and there is **no way** to do something wrong with that. 
Unless you start testing another class inside InMemoryRepositoryTest.class :D